src/btree.c

```diff
@@ -3145,100 +3145,104 @@ btree_count_in_range_by_iterator(cache        *cc,
    } else {
-      platform_log_stream("**  LEAF NODE \n");
-      platform_log_stream("**  addr: %lu \n", addr);
-      platform_log_stream("**  ptr: %p\n", hdr);
-      platform_log_stream("**  next_addr: %lu \n", hdr->next_addr);
-      platform_log_stream("**  next_extent_addr: %lu \n",
-                          hdr->next_extent_addr);
-      platform_log_stream("**  generation: %lu \n", hdr->generation);
-      platform_log_stream("**  height: %u \n", btree_height(hdr));
-      platform_log_stream("**  next_entry: %u \n", hdr->next_entry);
-      platform_log_stream("**  num_entries: %u \n", btree_num_entries(hdr));
-      platform_log_stream("-------------------\n");
+      platform_log(log_handle, "**  LEAF NODE \n");
+      platform_log(log_handle, "**  addr: %lu \n", addr);
+      platform_log(log_handle, "**  ptr: %p\n", hdr);
+      platform_log(log_handle, "**  next_addr: %lu \n", hdr->next_addr);
+      platform_log(
+         log_handle, "**  next_extent_addr: %lu \n", hdr->next_extent_addr);
+      platform_log(log_handle, "**  generation: %lu \n", hdr->generation);
+      platform_log(log_handle, "**  height: %u \n", btree_height(hdr));
+      platform_log(log_handle, "**  next_entry: %u \n", hdr->next_entry);
+      platform_log(
+         log_handle, "**  num_entries: %u \n", btree_num_entries(hdr));
+      platform_log(log_handle, "-------------------\n");
       for (uint64 i = 0; i < btree_num_entries(hdr); i++) {
-         platform_log_stream("%lu:%u ", i, btree_get_table_entry(hdr, i));
+         platform_log(log_handle, "%lu:%u ", i, btree_get_table_entry(hdr, i));
       }
-      platform_log_stream("\n");
-      platform_log_stream("-------------------\n");
+      platform_log(log_handle, "\n");
+      platform_log(log_handle, "-------------------\n");
       for (uint64 i = 0; i < btree_num_entries(hdr); i++) {
          leaf_entry *entry = btree_get_leaf_entry(cfg, hdr, i);
-         platform_log_stream(
-            "%2lu:%s -- %s\n",
-            i,
-            key_string(dcfg, leaf_entry_key_slice(entry)),
-            message_string(dcfg, leaf_entry_message_slice(entry)));
+         platform_log(log_handle,
+                      "%2lu:%s -- %s\n",
+                      i,
+                      key_string(dcfg, leaf_entry_key_slice(entry)),
+                      message_string(dcfg, leaf_entry_message_slice(entry)));
       }
-      platform_log_stream("-------------------\n");
-      platform_log_stream("\n");
+      platform_log(log_handle, "-------------------\n");
+      platform_log(log_handle, "\n");
    }
 }
 
 void
-btree_print_node(cache                 *cc,
-                 btree_config          *cfg,
-                 btree_node            *node,
-                 platform_stream_handle stream)
+btree_print_node(cache               *cc,
+                 btree_config        *cfg,
+                 btree_node          *node,
+                 platform_log_handle *log_handle)
```

Comment:
Here is another example where the `platform_log_handle *log_handle` could be moved to become the 1st arg.

Btw, I don't have a very strong reason to justify this, except that when I see diffs like this (somewhere else in this file):

```
-   btree_print_node(cc, cfg, &node, PLATFORM_DEFAULT_LOG_HANDLE);
+   btree_print_node(cc, cfg, &node, Platform_default_log_handle);
```

I wondered ... if we could take the opportunity to make the log-handle / stream as the 1st arg, while we are anyways over-hauling such lines of code. 

(If we don't do it now, we won't come back to change it one more time in the future.)